reliability/config/enhance_reliability.yaml

```diff
@@ -12,6 +12,13 @@ reliability:
     # for 5 nodes m5.xlarge cluster, 60 is recomended
     maxProjects: 25
     sleepTime: 10
+  cerberusIntegration:
+    # start cerberus https://github.com/cloud-bulldozer/cerberus before starting reliabiity test.
```

Comment:
While GitHub Repo for Cerberus remains the official source of starting the cerberus, here's a tiny script we can link to here in the repo to make it easier: 
```run_cerberus.sh 
#!/bin/sh
function getKubeconfigFromFlexy(){
curl --insecure https://<jenkins_url>/job/ocp-common/job/Flexy-install/$BUILD_ID/artifact/workdir/install-dir/auth/kubeconfig -o ~/.kube/config$BUILD_ID ; export KUBECONFIG=~/.kube/config$BUILD_ID


}

echo "Invocation as run_cerberus.sh <FlexybuildId> <hostportmapping>"
BUILD_ID=$1
PORT=$2
echo "Using build $BUILD_ID and port $PORT"
set -x
# getKubeconfigFromFlexy

sudo firewall-cmd --add-port=$PORT/tcp --permanent
sudo firewall-cmd --reload

podman logs -f `podman run --name=cerberus$BUILD_ID -e SLACK_API_TOKEN="<token>" -e SLACK_CHANNEL=G01195FMYLF -p $PORT:8080 -v ~/.kube/config$BUILD_ID:/root/.kube/config:Z -v ~/cerberus_config.yaml:/root/cerberus/config/config.yaml:Z -d quay.io/openshift-scale/cerberus:latest`
```


This will create and start tailing the logs for a cerberus container. Also expose ports through firewall.